### PR TITLE
Remove :host statements not needed

### PR DESCRIPTION
**Pull request by jeroach**
Step 1 of the Lockhart hit list :)

Addressing `CSS - Unnecessary :host prefixes` (Slide 5) of:
https://docs.google.com/presentation/d/1EhZJonzPZUb8kDPZDfqOyvadodO6171utv6c7lzY8kc/edit

### CODE REVIEW

**Comment by maboelsoudd2l** [comment]
Side note: The visual diffing would have been very useful in a PR like this to quickly check for any changes that the eye might miss by just glancing over.

**Comment by jeroach** [comment]
> Side note: The visual diffing would have been very useful in a PR like this to quickly check for any changes that the eye might miss by just glancing over.

100% agree - I did a visual check offline, but visual diff would be much easier

File: components/d2l-quick-eval/activity-card/d2l-quick-eval-activity-card-action-button.js [hunk 1]
```diff
@@ -87,15 +87,15 @@ class D2LQuickEvalActivityCardActionButton extends mixinBehaviors(
 						--d2l-quick-eval-card-button-icon-border-radius: .01rem;
 					}
 				}
-				:host([disabled]) button {
+				[disabled] button {
```
Comment (dlockhart) [inline components/d2l-quick-eval/activity-card/d2l-quick-eval-activity-card-action-button.js]: The `:host(whatever)` ones you may want to keep actually... I'm surprised this is working without it TBH.

Comment (jeroach) [reply components/d2l-quick-eval/activity-card/d2l-quick-eval-activity-card-action-button.js]: is this for the `rtl` stuff too?

Comment (jeroach) [reply components/d2l-quick-eval/activity-card/d2l-quick-eval-activity-card-action-button.js]: @dlockhart and I chatted offline, should be for RTL stuff too - `I just meant that it’s safe to remove :host something but I wasn’t recommending removing :host(this) something`

Comment (jeroach) [reply components/d2l-quick-eval/activity-card/d2l-quick-eval-activity-card-action-button.js]: I'm fixing now :)

**Pull request merged**